### DISCUSSION TITLE
Helping you writing Examples for Solidjs

### DISCUSSION THREAD

**Discussion by q1b** [TanStack/table#3911] [Q&A]
I am a front-end engineer, using solidjs and typescript as my main stack in my daily job, I know react as well, just want to ask Is it possible for me to add examples for solidjs, ( 🤪 JUST convert React code into Solidjs and make it work, ( also understand how react-table ( like solid-table ) library works, as well ) hope you like the idea ) Hope for a YES.

**Comment by KevinVandy** [comment]
Yes that would be very helpful. There's a directory in examples/solid for them.

**Reply by KevinVandy** [reply]
also, consider joining the Tanstack discord react-table-v8 channel if you want to be more a part of the public development discussion. Easier to coordinate there.

One of the largest priorities for the next month is creating more examples and docs for each supported framework, so your help would be very welcome

**Comment by tannerlinsley** [comment]
Sure!

Tanner Linsley

**Reply by Blm0423** [reply]
Hi Tanner,
I would appreciate examples for SolidJS of filtering (global and column) and OnRowClick.
Hopefully more people will have use for those as well.
Thanks//Jan
